openeo_udp/sentinel1_stats/sentinel1_stats.json

```diff
@@ -150,7 +150,7 @@
   },
   "id": "sentinel1_stats",
   "summary": "Calculate Sentinel-1 SAR stats",
-  "description": "Calculate S1 SAR stats (min, max, mean, sd, q10, q50, q90). \n\nThis service can cost an approximate of 3-5 credits per sq km. \nThis cost is based on resource consumption only and added-value cost has not been included.\n",
+  "description": "# Description\n\nThis algorithm derives temporal statistics from the Sentinel-1 GRD collection for the selected spatial- and temporal extent of interest. In order to calibrate the SAR backscatter signal, `sigma0-ellipsoid` has been applied within this workflow.\n\n# Performance characteristics\nThe algorithm was evaluated for a spatial extent of 20x20km, for a temporal period of 3 months. The total cost for the performed evaluation was equal to 4 credits, thereby highlighting the efficiency of the underlying algorithm.\n\n# Examples\n\nBelow we show the derived minimum and maximum values for both VV and VH bands, calculated for the 20x20 km2 area of interest.\n\n![s1_stats](./sentinel1.png)",
```

Comment:
note that the original text comes from https://github.com/ESA-APEx/apex_algorithms/blob/b2f7af26a231d88d47cbf09ce61be4088cb58ef3/openeo_udp/sentinel1_stats/README.md
through https://github.com/ESA-APEx/apex_algorithms/blob/b2f7af26a231d88d47cbf09ce61be4088cb58ef3/openeo_udp/sentinel1_stats/generate.py#L58

So it should be updated too

Reply:
I have updated the README.md on the last commit https://github.com/ESA-APEx/apex_algorithms/pull/54/commits/c50217eaaa93443e9ac856d74ac2e07515aaef82